### PR TITLE
Exposition

### PR DESCRIPTION
**Pull request by dfenjves**
@AnnJohn @jmburges @StevenNunez first pass. Please let me know about improvements...

### CODE REVIEW

**Comment by AnnJohn** [comment]
Assigned to @StevenNunez for review

File: README.md [hunk 1]
```diff
@@ -16,12 +14,50 @@
...
 
+>If to do were as easy as to know what were good to do, chapels had been churches and poor men's cottages princes' palaces. It is a good divine that follows his own instructions: I can easier teach twenty what were good to be done, than be one of the twenty to follow mine own teaching. The brain may devise laws for the blood, but a hot temper leaps o'er a cold decree: such a hare is madness the youth, to skip o'er the meshes of good counsel the cripple. But this reasoning is not in the fashion to choose me a husband. O me, the word 'choose!' I may neither choose whom I would nor refuse whom I dislike; so is the will of a living daughter curbed by the will of a dead father. Is it not hard, Nerissa, that I cannot choose one nor refuse none?
+
+Your window should look like this:
+![rubular setup](http://curriculum-content.s3.amazonaws.com/web-development/Regex/rubular_setup.png)
```
Comment (octosteve) [inline README.md:22]: You can link a permalink to your window here.

File: README.md [hunk 2]
```diff
@@ -16,12 +14,50 @@
...
+```
+/twenty/
+```
+
+![twenty regex](https://curriculum-content.s3.amazonaws.com/web-development/Regex/twenty.png)
+
+Notice that the pattern matches the two instances of "twenty" in the passage. Writing a series of letters or numbers in your regular expression will result in a search for exact matches of this pattern anywhere in the string. 
+
+### Metacharacters
+The real beauty of Regular Expressions is revealed in its use of metacharacters. Metacharacters allow you to use a pre-defined shorthand to match specific characters. For example, `\d` will match any digit in your text, and `\w` will match any word character (non-digit, non-punctuation). The 'Regex Quick Reference' at the bottom of Rubular shows metacharacters and patterns that you can use. Play around with these a little. Use `\W` (notice uppercasing) to match the non-word characters in your text.
```
Comment (octosteve) [inline README.md]: `\w` matches anything from this character class `[a-zA-Z0-9_]`, so letters, number, and underscore

File: README.md [hunk 3]
```diff
@@ -16,12 +14,50 @@
...
+We can actually shorten this in ruby using a Regex range:`/[a-j]/`
+
+`[0123456789]` becomes `[0-9]`
+
+### Example: Double Vowels
+
+There are many other metacharacters and ways of building patterns in regex, many of which you can refer in the Rubular quick reference guide. However, the best way to actually learn to use Regular Expressions is to practice building your own patterns. Let's look for instances in our text of two consecutive vowels. The longest way to do this is to hand code the different combinations of two vowels:`/aa|oo|ee|ii|uu|ae|ea/`. It's pretty tedious to hand code each of these combinations (I didn't finish). An improvement is to use two sets of square brackets with vowels, each one representing a single character: `/[aeiou][aeiou]/`. Our most efficient, however, is to use repetitions: `/[aeiou]{2}/` The curly braces surrounding mean that the pattern or character directly preceding it must repeat that number of times. As such, we're looking for a repeat of a vowel two times. As you can see, there are many ways to write a Regular Expression that does the same thing.
```
Comment (octosteve) [inline README.md]: `/[aeiou]{2}/` matches aa as well as ai. To match a previous vowel you'd have to do something like (WARNING: CRAZY) `/([aeiou])\1/` meaning, match the previous value that was captured.

**Comment by octosteve** [comment]
@dfenjves Made some notes! Fix those and we're good to go.

**Pull request merged**